config/locales/en.yml

```
@@ -589,7 +589,7 @@ en:
         balance_due: "Balance Due"
       destroy:
         has_associated_orders: "Delete failed: customer has associated orders with his shop"
-
+        has_associated_subscriptions: "Delete failed: customer has active associated subscriptions with shops"
```

Just thinking about the wording. One customer record is associated to one shop. And there is no other use of the word subscription apart from automated periodic orders with a shop. We can skip "with shops".

The word "associated" is very technical. If a customer "has" a subscription, we already say that there's a relationship. 

I propose:
```suggestion
        has_associated_subscriptions: "Delete failed: This customer has active subscriptions. Cancel them first."
```